### ISSUE TITLE
Vulkan: FlowMap pixilation artifacts

### ISSUE THREAD

**Issue by AlfonsoGrimaldo** [godotengine/godot#67961]
### Godot version

v4.0.beta3.official [01ae26d31]

### System information

Windows 10, Vulkan, GTX 960 522.25

### Issue description

The picture describes everything:
![screenshot](https://user-images.githubusercontent.com/48926464/198414681-fdb24469-abe8-4a4f-b0a3-71b60bc2eb86.png)

### Steps to reproduce

1.- Create a Standard material.
2.-Set the roughness about 0.5.
3.- Activate the anisotropy and set it to 1.
4.- Insert the flowmap and observe.

### Minimal reproduction project

You can try it with this flowmap or with any other:
![anisodir01kmozm](https://user-images.githubusercontent.com/48926464/198415545-1cbf4abf-8ad4-4ce8-a9b4-b4f328b10df9.png)

**Comment by Calinou** [comment]
This is due to VRAM compression. Change the texture's import mode to **Lossless** to resolve this issue.

I wonder if flowmaps could be detected as normal maps to use RGTC compression. The texture's blue channel is ignored, so it can be safely discarded.

**Comment by AlfonsoGrimaldo** [comment]
@Calinou Yes, it's the compression. I supposedly checked this but I see that no.

**Comment by BlueCube3310** [comment]
In 4.5-dev4:
| BC1 / DXT1 | BC5 / RGTC_RG | BC7 / BPTC_RGBA |
|--------------|------------------|---------------------|
| ![Image](https://github.com/user-attachments/assets/876ef717-94bc-46db-8cea-6b817d3a79ec) | ![Image](https://github.com/user-attachments/assets/c09e507a-c3c7-4cc8-9266-728942bc195b) | ![Image](https://github.com/user-attachments/assets/e6b5dbc1-b576-498f-92e6-377fa8f10642) |

It looks slightly better, the artifacts only appear at the edges of the circle, though it's to be expected due to how block compression works.
